README.md

```diff
@@ -65,6 +65,11 @@ Configuration is achieved through environment variables.
 | Environment Variable   | Supported Values | Notes             |
 | :--------------------- | :--------------- | :---------------- |
 | `FTL_HOSTNAME`         | Valid hostname   | The hostname of the machine running the FTL service. Defaults to system hostname. |
+| `FTL_NODE_KIND`        | `Standalone`: (default) This instance will listen for incoming FTL connections and transmit them to WebRTC clients.<br />`Ingest`: This instance will listen for incoming FTL connections and relay them to other nodes when instructed by an Orchestrator service.<br />`Edge`: This instance will receive stream relays from other nodes and transmit them to WebRTC clients. | This configuration value controls the behavior of the FTL plugin when used in conjunction with an [Orchestrator service](https://github.com/Glimesh/janus-ftl-orchestrator). |
```

Comment:
Uppercased first letter for the values?

Reply:
They're case insensitive when processed, so it doesn't really matter.